skimage/filters/_gaussian.py

```diff
@@ -9,15 +9,15 @@
 __all__ = ['gaussian']
 
 
-def gaussian(image, sigma, output=None, mode='nearest', cval=0,
+def gaussian(image, sigma=1, output=None, mode='nearest', cval=0,
              multichannel=None):
     """Multi-dimensional Gaussian filter
 
     Parameters
     ----------
     image : array-like
         input image (grayscale or color) to filter.
```

Comment:
Not related to this PR, but capitalize "Input", same for "Standard" two lines down.